src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -57,6 +60,12 @@ public class PolicyService : IPolicyService
                         {
                             throw new BadRequestException("Single Sign-On Authentication policy is enabled.");
                         }
+
+                        var ssoConfig = await _ssoConfigRepository.GetByOrganizationIdAsync(org.Id);
+                        if (ssoConfig?.GetData()?.UseKeyConnector == true)
+                        {
+                            throw new BadRequestException("KeyConnector is enabled.");
+                        }
```

Comment:
Lets move this logic to a private method.

Reply:
Duplicate of above.